src/com/content-stats/stats-event.js

```diff
@@ -77,31 +80,61 @@ export default class ContentStatsEvent extends Component {
 
 			var Data = [];
 			if ( EventMode ) {
-				Data.push(<div><span class="-title">Signups:</span> <span class="-value">{stats.signups}</span></div>);
-				Data.push(<div><span class="-title">Unique Authors:</span> <span class="-value">{stats.authors}</span></div>);
+				Data.push(<div class="-gap"><span class="-title">Number of Users:</span></div>);
```

Comment:
A rather smallish thing, but I would prefer these three lines to be a single JSX (i.e. have outer div) and be assigned to a variable with a helping name.

`ShowIntroStats = <Common>your lines </Common>;`

I'm not sure exactly what of the already defined components are most correct but should be easy to spy on other files. 

Then at the top you would have some `let ShowIntroStats = null`

This is at least the general pattern I've seen in the other ui files.

Using arrays feels like it should be reserved for preparing a list of near identical things that could be done in a `forEach` or similar.

Reply:
Heh, the pushes initially were actually me. :laughing:

This file was written in a hurry, and each line is basically a copy+paste with a different data field as I exposed them with the API. I was more concerned with roughing it in than anything. There was no rhyme or reason. I just kept adding new stats as I had them.

Reply:
On a side node, though I don't mind the style of the `let ShowBlah = <div />;`, I can't say I'm 100% happy with the word `Show`. I've been thinking the same thing any time I've seen a member variable named `Class` (first character in caps), thinking it should be `Classes` or something else. Anyway, not really related to this issue.

Reply:
I think the array is the cleanest way to do it in this case, otherwise we would have a dozen separate variables.

Reply:
I don't agree, as far as I see it about 5 or less are needed.

```
ShowSingups
ShowCategory
ShowVoting
```

Each would correspond to several of current `push` since they are logical blocks anyhow and could just as well have a div around themselves.